src/sentry/seer/anomaly_detection/types.py

```diff
@@ -0,0 +1,25 @@
+from typing import TypedDict
+
+
+class Alert(TypedDict):
```

Comment:
re: above

can we name this something more descriptive and less saturated?

Reply:
again no, this matches the data in the Seer repo